server/portal/apps/datafiles/views.py

```diff
@@ -38,6 +38,15 @@ def get(self, request):
         default_system = user_systems[settings.PORTAL_DATA_DEPOT_LOCAL_STORAGE_SYSTEM_DEFAULT]
         response['default_host'] = default_system['host']
 
+        for system in response['system_list']:
+            try:
+                if system['api'] == 'tapis':
```

Comment:
I know these extra calls to systems and how they could affect the delay to this call.  could we add a JIRA ticket for that?

what about also about adding a check for `viewPath` in `WORKBENCH_SETTINGS` in this `if` in the meantime?